ReleaseProcessAutomation.UnitTests/Steps/Releases/ReleaseNonPreReleaseFromDevelopTests.cs

```diff
@@ -51,6 +52,7 @@ public void Setup ()
     _continueReleaseOnMasterMock = new Mock<IContinueReleaseOnMasterStep>();
     _consoleMock = new Mock<IAnsiConsole>();
     _jiraFunctionalityMock = new Mock<IJiraFunctionality>();
+    _pushNewReleaseBranchMock = new Mock<IPushNewReleaseBranchStep>();
```

Comment:
I just realized that using a loose mock here is kind of bad. If the step is called when we don't want it to be called then such a test would not fail. We should use a strict mock and explicitly setup when we expect the call.